packages/turbo-utils/__tests__/logger.test.ts

```diff
@@ -0,0 +1,210 @@
+import {
+  describe,
+  test,
+  expect,
+  beforeEach,
+  afterEach,
+  jest,
+} from "@jest/globals";
+import {
+  turboBlue,
+  turboRed,
+  yellow,
+  turboLoader,
+  info,
+  bold,
+  underline,
+  dimmed,
+  grey,
+  item,
+  log,
+  warn,
+  error,
+} from "../src/logger";
+
+// Mock console methods
+const mockConsoleLog = jest.fn();
+const mockConsoleError = jest.fn();
+
+beforeEach(() => {
+  jest.clearAllMocks();
+  jest.spyOn(console, "log").mockImplementation(mockConsoleLog);
+  jest.spyOn(console, "error").mockImplementation(mockConsoleError);
+});
+
+afterEach(() => {
+  jest.restoreAllMocks();
+});
+
+describe("logger utilities", () => {
+  describe("color functions", () => {
```

Comment:
This is basically us testing `picocolors` so not super useful

Reply:
Trust no one. /s